src/State/Credits.cpp

```diff
@@ -57,8 +57,7 @@ void Credits::init()
 
     Game::getInstance()->mouse()->pushState(Input::Mouse::Cursor::NONE);
     auto renderer = Game::getInstance()->renderer();
-    setX((renderer->width()  - 640)*0.5);
-    setY(renderer->height());
+    setPosition(Point((renderer->size().width() - 640) / 2, renderer->size().height()));
```

Comment:
This is because we need to divide only X portion of position here... If you have a better idea, please elaborate.